config.yml

```diff
@@ -33,8 +33,9 @@ native_service_types:
   - gcs
   - abs
 
-# some id
-#connector_id: '1'
+# Connector client settings
```

Comment:
maybe we can add an extra comment to say that when used on-premise, we need both (it's not explicit in Kibana's UI)

Reply:
Ideally, we should have some validation. There are many places in the code where we just assume a valid config. This is the same for the ruby connector where we only have basic validation (e.g. a field is required and it must be an integer)